frontend/src/action/player-actions.js

```diff
@@ -0,0 +1,74 @@
+import superagent from 'superagent';
+
+export const playerSet = players => {
+  return {
+    type: 'PLAYER_SET',
+    payload: players,
+  };
+};
+
+export const playerCreate = player => {
+  return {
+    type: 'PLAYER_CREATE',
+    payload: player,
+  };
+};
+
+export const playerUpdate = player => {
+  return {
+    type: 'PLAYER_UPDATE',
+    payload: player,
+  };
+};
+
+export const playerDestroy = player => {
+  return {
+    type: 'PLAYER_DESTROY',
+    payload: player,
+  };
+};
+
+/***********************************
+*     ASYNC ACTIONS     *
+************************************/
+export const playersFetchAll = () => dispatch => {
+  return superagent.get(`http://localhost:3000/api/v1/players`)
```

Comment:
Pull localhost:3000 into a global variable like __API_URL__ in the future. You could even include the /api/v1/ in the global depending on implementation. In this case it seems like that would be appropriate and could be changed later if circumstances change.